### PR TITLE
[DEV APPROVED] 9298 site margins

### PR DESCRIPTION
**Pull request by rhianwatkins1**
[TP9298](https://moneyadviceservice.tpondemand.com/entity/9298-apply-consistent-margins)

This PR is a big bit of work tidying up the margins and spacing used in the site to be more consistent, and less dependant on using lots of general styling classes in the markup.

Also updated the grids to go to the edge of it's container, removing the extra spacing wherever they were used.

### CODE REVIEW

**Comment by masbuild** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">There are code changes, but no corresponding tests. Please include tests if this PR introduces any modifications in behavior. If this is a refactor PR, please ignore the message.Refer to our mas standards guides about opening a PR<a href="link-to-the-mas-standards-about-opening-a-pr">here</a></td>
    </tr>
  </tbody>
</table>

At Money Advice Service, we use Danger gem, to ensure that our
commits and pull requests follow  [standards](https://github.com/moneyadviceservice/mas-standards/blob/master/Dangerfile).
If you think one of the points here has been raised in error (or   shouldn't be in our standards), open an issue or pull request on [our   Danger config](https://github.com/moneyadviceservice/mas-standards).


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: app/views/fincap_templates/index.html.erb [hunk 1]
```diff
@@ -4,31 +4,25 @@
...
+    <div class='teaser-box--thematic-review'>
+      <img class='teaser-box__image'
+             src=<%= template.hero_image_component %>
+             alt=''
+        />
+      <div class='teaser-box__content'>
+        <h3 class='teaser-box__title'>Thematic Review:</h3>
+        <a href= "<%= template.full_path %>" class='teaser-box__content-link'>
+          <%= template.title %>
+        </a>
```
Comment (tomas-stefano) [inline app/views/fincap_templates/index.html.erb]: Can you change this line to: `<%= link_to template.title, template.full_path, class: "teaser-box__content-link"  %>`

File: app/views/fincap_templates/index.html.erb [hunk 2]
```diff
@@ -4,31 +4,25 @@
...
-            <p class='teaser-box__content-text'>
-              <%= template.overview_content.html_safe %>
-            </p>
-          </div>
-        </div>
+    <div class='teaser-box--thematic-review'>
+      <img class='teaser-box__image'
+             src=<%= template.hero_image_component %>
+             alt=''
+        />
```
Comment (tomas-stefano) [inline app/views/fincap_templates/index.html.erb]: Can you change this line to: `<%= image_tag template.hero_image_component, class: 'teaser-box__image' %>`

**Comment by tomas-stefano** [comment]
If you don't mind I renamed to WIP. 🙏

**Pull request merged**